packages/ui/.storybook/main.js

```diff
@@ -1,13 +1,15 @@
 module.exports = {
-  stories: [
-    '../src/**/*.stories.@(js|jsx|mdx|ts|tsx)',
-    './**/*.stories.@(js|jsx|mdx|ts|tsx)',
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|mdx|ts|tsx)"
```

Comment:
isn't this line including the previous one already?

Reply:
yep, I'll fix it